conbench/tests/api/test_compare.py

```diff
@@ -59,7 +59,16 @@ def test_compare(self, client):
             run_ids,
             "read",
             CASE,
+            tags={
+                "dataset": "nyctaxi_sample",
+                "cpu_count": 2,
+                "file_type": "parquet",
+                "input_type": "arrow",
+                "compression": "snappy",
+                "name": "read",
+            },
         )
+        # import pdb; pdb.set_trace()
```

Comment:
Remove pdb

Reply:
Ack, I thought I had caught all of these already!